pkg/apis/pipeline/v1/pipeline_types_test.go

```diff
@@ -668,93 +668,37 @@ func TestPipelineTask_ValidateMatrix(t *testing.T) {
 			Paths:   []string{"matrix.include[0].params[1].name"},
 		},
 	}, {
-		name: "parameters in matrix are strings",
+		name: "parameters in matrix contain references to arrays",
```

Comment:
this test is changed in v1, but removed in v1beta1, please sync

Reply:
This is in v1beta1
https://github.com/tektoncd/pipeline/blob/7dedfc68ea9427c49c7b8d767214fce9d4f8bfb2/pkg/apis/pipeline/v1beta1/pipeline_types_test.go#L645